docs/source/conceptual/visual_language_models.md

```diff
@@ -0,0 +1,224 @@
+# Vision Language Models (VLM)
+
+## What is VLM?
+
+Visual Language Model (VLM) are models that consume both visual and textual inputs to generate text.
+
+These models are trained on multimodal data, which includes both images and text.
```

Comment:
would be better not to break the paragraph because you are referring to VLMs with _these_ and the context is the same anyway, this breaks the readability

Reply:
paragraph rewritten and is not broken into two sentences in the latest changes